### PR TITLE
tts: 0.1.2 -> 0.1.3

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/coqui-ai/TTS/releases/tag/v0.1.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/audio/tts/default.nix [hunk 1]
```diff
@@ -11,25 +11,20 @@
...
-  # https://github.com/coqui-ai/TTS/issues/570
-  disabled = python3.pythonAtLeast "3.9";
+  version = "0.1.3";
 
   src = fetchFromGitHub {
     owner = "coqui-ai";
     repo = "TTS";
     rev = "v${version}";
-    sha256 = "1qgiaqn7iqxyf54qgnpmli69nw9s3gmi9qv874jsgycykc10hjg4";
+    sha256 = "sha256-fl4nrBCznalzc8wu+30rgORID9TW7tvJikqik7TYjp4=";
```
Comment (Mic92) [inline pkgs/tools/audio/tts/default.nix:27]: nix-shell:~/.cache/nixpkgs-review/pr-131598]$ nix-shell -p tts
these 2 derivations will be built:
  /nix/store/x1vbjrii5cj53bb6qyrs62v9ba521rdm-source.drv
  /nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv
warning: unknown setting 'experimental-features'
building '/nix/store/x1vbjrii5cj53bb6qyrs62v9ba521rdm-source.drv'...


trying https://github.com/coqui-ai/TTS/archive/v0.1.3.tar.gz
trying https://github.com/coqui-ai/TTS/archive/v0.1.3.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
                                 Dload  Upload   Total   Spent    Left  Speed
100   120    0   120    0     0    446      0 --:--:-- --:--:-- --:--:--   444
100   120    0   120    0     0    446      0 --:--:-- --:--:-- --:--:--   444
100 13.9M    0 13.9M    0     0  9256k      0 --:--:--  0:00:01 --:--:-- 15.6M
100 13.9M    0 13.9M    0     0  9256k      0 --:--:--  0:00:01 --:--:-- 15.6M
unpacking source archive /build/v0.1.3.tar.gz
unpacking source archive /build/v0.1.3.tar.gz
hash mismatch in fixed-output derivation '/nix/store/0lwp3b1k7w00hk3kygf6mg1njkb85vrg-source':
  wanted: sha256:17lfv2s978jaib4xpvnnsh7lir405dyznbncfdrsk7dk22n2fpky
  got:    sha256:0akhiaaqz53bf5zyps3vgjifmgh5wvcc9r4lrq9hmj3dds03vkjq
cannot build derivation '/nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv': 1 dependencies couldn't be built
error: build of '/nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv' failed

Did they maybe changed the release again?

Comment (mweinelt) [reply pkgs/tools/audio/tts/default.nix]: Possibly, I'm not sure. Pushed an update.

File: pkgs/tools/audio/tts/default.nix [hunk 2]
```diff
@@ -10,26 +10,19 @@
 # $ tts-server --model_name tts_models/en/ljspeech/glow-tts --vocoder_name vocoder_models/universal/libri-tts/fullband-melgan
 #
 # If you upgrade from an old version you may have to delete old models from ~/.local/share/tts
-# Also note that your tts version might not support all available models so check:
-#   https://github.com/coqui-ai/TTS/releases/tag/v0.1.2
```
Comment (mweinelt) [inline pkgs/tools/audio/tts/default.nix:14]: The release notes don't contain model hints anymore.

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 131598` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tts</li>
  </ul>
</details>

**Review by davidak — APPROVED** [review]
Change looks good

Builds, works

**Pull request merged**